### PR TITLE
Fix #81: Be explicit in oppiabot's comments

### LINKED ISSUE #81: Be more explicit about the comment on disallowed branches
Oppiabot should be more explicit in it's comment on PRs made from test, develop or release branches.
https://github.com/oppia/oppia/pull/9513#issuecomment-643580955

### PR DESCRIPTION
**Pull request by jameesjohn**
## Explanation
Fixes #81: Be explicit in oppiabot's comments in branch check.

## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: lib/checkPullRequestBranch.js [hunk 1]
```diff
@@ -36,10 +36,11 @@ const checkBranch = async (context) => {
...
       body:
-        'Hi @' + prAuthor + ', PRs made from develop, release or test ' +
-        'branches are not allowed. So this PR is being closed. Please make ' +
-        'your changes in another branch and send in the PR. To learn more ' +
-        'about contributing to Oppia, take a look at our ' + wiki + '. Thanks!'
+        'Hi @' + prAuthor + ', PRs made from develop branch or prefixed with ' +
+        'release- or test- are not allowed. So this PR is being closed. ' +
+        'Please make your changes in another branch and send in the PR. To ' +
+        'learn more about contributing to Oppia, take a look at our '
+        + wiki + '. Thanks!'
```
Comment (ankita240796) [inline lib/checkPullRequestBranch.js:43]: Change to:

        'Hi @' + prAuthor + ', PRs made from develop branch or from a branch whose name is prefixed with ' +
        'develop, release or test are not allowed. So this PR is being closed. ' +
        'Please make your changes in another branch and send in the PR. To ' +
        'learn more about contributing to Oppia, take a look at our '
        + wiki + '. (Rule 1 specifically) Thanks!'

Comment (jameesjohn) [reply lib/checkPullRequestBranch.js]: Done.
Thanks!

File: lib/checkPullRequestBranch.js [hunk 2]
```diff
@@ -36,10 +36,12 @@ const checkBranch = async (context) => {
...
-        'Hi @' + prAuthor + ', PRs made from develop, release or test ' +
-        'branches are not allowed. So this PR is being closed. Please make ' +
-        'your changes in another branch and send in the PR. To learn more ' +
-        'about contributing to Oppia, take a look at our ' + wiki + '. Thanks!'
+        'Hi @' + prAuthor + ', PRs made from develop branch or from a branch ' +
+        'whose name is prefixed with develop, release or test are not ' +
+        'allowed. So this PR is being closed. Please make your changes in ' +
+        'another branch and send in the PR. To learn more about contributing ' +
+        'to Oppia, take a look at our ' + wiki +
+        '. (Rule 1 specifically) Thanks!'
```
Comment (ankita240796) [inline lib/checkPullRequestBranch.js:44]: Full stop after the rule 1 bracket.

Comment (jameesjohn) [reply lib/checkPullRequestBranch.js]: Done.

File: lib/checkPullRequestBranch.js [hunk 3]
```diff
@@ -32,14 +32,20 @@ const checkBranch = async (context) => {
...
-        'about contributing to Oppia, take a look at our ' + wiki + '. Thanks!'
+        'Hi @' +
+        prAuthor +
+        ', PRs made from develop branch or from a branch ' +
+        'whose name is prefixed with develop, release or test are not ' +
+        'allowed. So this PR is being closed. Please make your changes in ' +
+        'another branch and send in the PR. To learn more about contributing ' +
+        'to Oppia, take a look at our ' +
+        wiki +
+        '. (Rule 1 specifically). Thanks!',
```
Comment (ankita240796) [inline lib/checkPullRequestBranch.js:48]: Remove the fullstop before, it isn't required, right?

Comment (jameesjohn) [reply lib/checkPullRequestBranch.js]: Nope not needed.
Removed.

**Pull request merged**